lib/transaction.js

```diff
@@ -469,13 +536,19 @@ export default class Transaction {
   /* eslint-enable prefer-destructuring */
 
 
-  toJSON() {
+  toJSON(prevTx = null) {
     const json = {
       type: this.type,
       inputs: this.inputs.map(inp => inp.toJSON()),
       outputs: this.outputs.map(out => out.toJSON()),
     };
 
+    if (this.type === Type.TRANSFER) {
+      json.to = this.to();
+      json.from = this.from(prevTx);
+      json.value = this.value(prevTx);
```

Comment:
I would split this into two separate keys: `value` and `color`. We have such format in the node JSON RPC (getTransaction)